### ISSUE TITLE
[Question] Does myDrive offer end-to-end encryption?

### ISSUE THREAD

**Issue by mdPlusPlus** [subnub/myDrive#62]
I'm struggling to understand what exactly [this segment](https://github.com/subnub/myDrive#webui-for-encryption-key) of the README is trying to tell me.

Does myDrive offer end-to-end encryption?

**Comment by subnub** [comment]
No, myDrive stores the encryption keys server side, it is **not** end-to-end encrypted.
